server/database/queries/authQueries.js

```diff
@@ -0,0 +1,13 @@
+const connection = require('../config/connection');
+
+const signUpQuery = ({ user_name, email, password }) => {
+  const sql = {
+    text: 'INSERT INTO users (user_name,email,password) VALUES ($1,$2,$3)  Returning *',
```

Comment:
you don't have to return the entire row just return the attributes you use
```suggestion
    text: 'INSERT INTO users (user_name,email,password) VALUES ($1,$2,$3)  Returning id, user_name, email',
```